lib/model/graph.js

```diff
@@ -1,46 +1,43 @@
-/**
- * graph.js
- * @author John O'Brien
- *
- * Definition for the Graph class. The Graph class holds on to every node
- * and connection in the entire graph.
- */
+var createClass = require('./../utilities/createClass');
+var GraphModelBase = require('./graphModelBase');
+var AttrValidators = require('./../utilities/attrValidators');
 
-/**
- * Creates a new graph.
- * @constructor
- *
- * @param nodes - An array containing node objects.
- * @param connections - An array containing connection objects for just
- *                      these tops level nodes.
- */
-var Graph = function(nodes, connections) {
-  this._nodes = nodes || [];
-  this._connections = connections || [];
+var Node = require('./node');
```

Comment:
I will resolve conflicts when this branch diverges from develop. There will be a lot of them.